tsp-output/@azure-tools/typespec-csharp/azoai/AzureOpenAI/Models/AzureChatExtensionConfiguration.Serialization.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.ClientModel.Primitives;
+using System.Text.Json;
+
+namespace AzureOpenAI.Models;
+
+[PersistableModelProxy(typeof(UnknownAzureChatExtensionConfiguration))]
+public partial class AzureChatExtensionConfiguration : IJsonModel<AzureChatExtensionConfiguration>
+{
+    void IJsonModel<AzureChatExtensionConfiguration>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
```

Comment:
Per @KrzysztofCwalina, look at different options for "Wire" and "AzureWire"

Reply:
We decided to move away from this approach in favor of having separate models for each type.